tomcat8-maven-plugin/src/main/java/org/apache/tomcat/maven/plugin/tomcat8/run/RunMojo.java

```diff
@@ -469,21 +469,20 @@ private WebResource urlToWebResource( URL url, String path )
 
                         try
                         {
-                            // url.getFile is
-                            // file:/Users/olamy/mvn-repo/org/springframework/spring-web/4.0.0.RELEASE/spring-web-4.0.0.RELEASE.jar!/org/springframework/web/context/ContextLoaderListener.class
-
-                            int idx = url.getFile().indexOf( '!' );
+                            URI uri = new URI(url.getFile());
```

Comment:
the example comment was helpful... any reason to delete it? maybe just clarify it as an example.

Reply:
Good point, I re-added that in. I originally had another implementation that didn't use that method so that is why I removed it.